bank/README.md

```diff
@@ -0,0 +1,411 @@
+2024-06-27 14:15:23 [info]:     POST    :  /api/withdraw 
+2024-06-27 14:15:23 [info]:         Required Roles: ["appUser"] 
+2024-06-27 14:15:23 [info]:     POST    :  /api/transfer 
+2024-06-27 14:15:23 [info]:         Required Roles: ["appUser"] 
+2024-06-27 14:15:23 [info]:     GET     :  /api/transaction_history/:accountId 
+2024-06-27 14:15:23 [info]:         Required Roles: ["appUser"] 
+```
+
+If you are interested in learning more about declarative security in DBOS, please read our [Authentication and Authorization](https://docs.dbos.dev/tutorials/authentication-authorization) tutorial.
+
+## Deploying to DBOS Cloud
+We can make some quick changes to deploy the DBOS Bank backend in the cloud, using the free "DBOS Starter" hosting tier.
+
+### Registration / Login
+First, from the `bank-backend/` directory, execute `npx dbos-cloud register -u <username>` to register an account with DBOS cloud, or, if you are already registered execute `npx dbos-cloud login`.  This will allow you to log in securely in your browser, or create an account if you do not already have one.
+
+### Creating Cloud Database
+After logging in, create a Postgres server using `npx dbos-cloud database provision --username <pg db admin username> dbosbankdb`, changing the username and password as desired.  You will be prompted for the database password.
+
+### Deploying DBOS Bank Backend
+Deploying the first bank is straightforward:
+- Register the bank app with `npx dbos-cloud app register -d dbosbankdb bank_a`.
+- Deploy the app with `CURRENT_BANK=bank_a npx dbos-cloud app deploy bank_a`.
+- Make a note of the `Access your application at` URL displayed, as this will be necessary for configuring the frontend.
+
+Deploying the second bank similarly:
+- Register the bank app with `npx dbos-cloud app register -d dbosbankdb bank_b`.
+- Deploy the app with `CURRENT_BANK=bank_b npx dbos-cloud app deploy bank_b`.
+- Make a note of the `Access your application at` URL displayed, as this will be necessary for configuring the frontend.
+
+### Configuring and Launching Frontend
+
+The frontend AngularJS is not hosted in DBOS Cloud; we will just continue to use the local server for the demo.
+
+Edit the settings in `bank-frontend/src/environments/environment.ts` so that the frontend can find the bank servers.  Be sure to change both `bankHosts` and the `authUrl`, using the URLs provided from the deployment above.
+
+After configuration, you should be able to start the frontend from the `bank-frontend` directory as before:
+```bash
+npm install
+npm start
+```
+
+### Monitoring and Debugging
+
+DBOS Cloud comes with a [monitoring dashboard](https://docs.dbos.dev/cloud-tutorials/monitoring-dashboard) automatically.  To find the URL for your dashboard, execute `npx dbos-cloud dashboard url`.  The dashboard includes execution traces and the ability to run workflows under the [Time Travel Debugger](https://docs.dbos.dev/cloud-tutorials/timetravel-debugging).
```

Comment:
Unfortunately we don't support time travel debugging for Prisma apps. We do support interactive time travel for it. https://docs.dbos.dev/cloud-tutorials/interactive-timetravel

Reply:
That'd be what I get for not actually trying it.

Reply:
I just left debugger out for now.  We can go back.